stdlib/public/core/ArrayBuffer.swift

```diff
@@ -120,7 +120,10 @@ extension _ArrayBuffer {
     if !_isClassOrObjCExistential(Element.self) {
       return _storage.isUniquelyReferenced_native_noSpareBits()
     }
-    return _storage.isUniquelyReferencedNative() && _isNative
+    if !_storage.isUniquelyReferencedNative() {
+      return false
+    }
+    return _isNative
```

Comment:
@gottesmm Thanks for fixing, but we need to place comments the standard library code wherever we use specific non-idiomatic workarounds for serious performance issues!

Reply:
Fixed.

Reply:
🥇